src/editors/data/redux/app/reducer.test.js

```diff
@@ -0,0 +1,59 @@
+import { initialState, actions, reducer } from './reducer';
+
+const testingState = {
+  ...initialState,
+  arbitraryField: 'arbitrary',
+};
+
+describe('requests reducer', () => {
```

Comment:
this is app reducer, not request reducer-- just commenting because I saw this, it is just a nit!

Reply:
```suggestion
describe('app reducer', () => {
```